sirbot_pyslackers/endpoints/slack/commands.py

```diff
@@ -255,3 +254,19 @@ async def resources(command, app):
     )
 
     await app.plugins["slack"].api.query(url=methods.CHAT_POST_MESSAGE, data=response)
+
+
+async def xpost(command, app):
+    slack = app.plugins["slack"]
+    response = Message()
+    response["channel"] = command["channel_id"]
+    response["unfurl_links"] = False
+
+    response["text"] = (
+        "Please don't cross post the same question in multiple channels."
```

Comment:
I think you will need to manually add spaces to this, otherwise the words that are in different text blocks will run-on each other. e.g `multiple channels.There`